### PR TITLE
Adding sitewide alerts logic.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Adds alerts to be read and dismissed by user.

## Acceptance criteria
- [ ] When adding an alert block in drupal, alert should show on corresponding page or sitewide

### CODE REVIEW

File: src/platform/site-wide/sass/merger.scss [hunk 1]
```diff
@@ -614,20 +608,74 @@ img[data-srcset] {
 
 .usa-alert-full-width {
   background-color: $color-gray-lightest;
+  display: none;
```
Comment (ncksllvn) [inline src/platform/site-wide/sass/merger.scss]: We already use `.usa-alert-full-width` alerts for displaying homepage banners, such as for the recent government shutdown. I'm afraid this is going to hide those. I think we need to add another class for Drupal-rendered dismissible alerts. Or possibly, alerts rendered from Drupal can have inline style applied to hide them.

File: src/platform/site-wide/alerts-dismiss-view.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  let l;
+  // Iterate through the wraps, and show.
+  for (l = 0; l < wrapsShower.length; l++) {
+    if (!wrapsShower[l].classList.contains('dismissed')) {
+      wrapsShower[l].classList.add('show-alert');
+    }
+  }
+}
+
+document.addEventListener('DOMContentLoaded', alertsBuildShow);
```
Comment (ncksllvn) [inline src/platform/site-wide/alerts-dismiss-view.js]: I think this fits better in our [Static Pages bundle](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/static-pages/static-pages-entry.js) instead of site-wide, because React applications will have their own methods of showing alerts.

File: src/platform/site-wide/sass/merger.scss (hunk 1 above)
Comment (ethanteague) [reply src/platform/site-wide/sass/merger.scss]: @ncksllvn :+1:

File: src/platform/site-wide/sass/merger.scss [hunk 3]
```diff
@@ -614,20 +608,74 @@ img[data-srcset] {
...
+    border-radius: 25px;
+    position: absolute;
+    right: 0;
+    top: 5px;
+    cursor: pointer;
+    &:after {
+      color: white;
+      content: "✖";
+    }
+  }
```
[the remark this answers is not in the document]
Comment (ethanteague) [reply src/platform/site-wide/sass/merger.scss]: @ncksllvn :+1:

File: src/platform/site-wide/alerts-dismiss-view.js (hunk 2 above)
Comment (ethanteague) [reply src/platform/site-wide/alerts-dismiss-view.js]: @ncksllvn :+1:

File: src/site/layouts/landing_page.drupal.liquid [hunk 4]
```diff
@@ -9,43 +10,46 @@
...
-              {% for spoke in fieldSpokes %}
-                <li>
-                    <a href="#{% if spoke.entity.fieldTitle != empty %}{{ spoke.entity.fieldTitle | hashReference: title }}{% endif %}">
-                        {{ spoke.entity.fieldTitle }}
-                    </a>
-                </li>
-              {% endfor %}
-            </ul>
-          {% endif %}
+        {% if fieldSpokes != empty %}
```
Comment (acabouet) [inline src/site/layouts/landing_page.drupal.liquid]: Not a huge deal but is it possible to revert the changes to indentations in this template? It looks like you just meant to include the alert.

**Comment by ethanteague** [comment]
Sure. Done.

File: src/platform/site-wide/index.js [hunk 5]
```diff
@@ -6,6 +6,7 @@ import '../monitoring/sentry.js';
 import './legacy/menu'; // Used in the footer.
 import './moment-setup';
 import './popups';
+import '../../applications/static-pages/alerts-dismiss-view';
```
Comment (ncksllvn) [inline src/platform/site-wide/index.js]: Can we move this import to the [static-pages-entry](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/static-pages/static-pages-entry.js#L21)?

**Review by ncksllvn — COMMENTED** [review]
One more change suggested! Looking good though.

File: src/platform/site-wide/index.js (hunk 5 above)
Comment (ethanteague) [reply src/platform/site-wide/index.js]: Sure thing, @ncksllvn - done.

**Review by ncksllvn — APPROVED** [review]
LGTM, will merge once CI checks out 👌
